### PR TITLE
Feat/tak 20 implementing the atoms

### PR DESCRIPTION
**Pull request by thunderTheGun**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -5,24 +5,24 @@
     },
     "extends": [
         "plugin:react/recommended",
-        "standard-with-typescript",
-        "plugin:@typescript-eslint/recommended",
-        "eslint:recommended"
-    ],
-    "overrides": [
+        "airbnb",
```
Comment (mikitovicsmark) [inline .eslintrc.json:8]: Generally it is a nice habit to create "atomic" PRs. If these chores e.g. editing configs are related to your current task, include it in the PR, but otherwise it is easier to review if you split it into a separate PR.

File: pages/slice-simulator.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { SliceSimulator } from '@prismicio/slice-simulator-react'
 import { SliceZone } from '@prismicio/react'
 
-import { components } from '../slices'
+// import { components } from '../slices.'
```
Comment (mikitovicsmark) [inline pages/slice-simulator.jsx]: Please delete it if it is unnecessary, or add further explanation why you've commented this out.

File: components/atoms/typography.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
```
Comment (mikitovicsmark) [inline components/atoms/typography.tsx]: It feels like that these files are duplicated. Are you generating these somehow ?

If I have files that purely have styled components i like to differentiate them with a naming convention.
I would usually do something like typography.styles.tsx - that way for me it is easier to search for a given file.

File: components/atoms/typography.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        font-family: 'Figtree', sans-serif;
+        font-style: normal;
+        font-weight: 600;
+        font-size: 64px;
+        line-height: 120%;
+        color: ${Green050};
+    `
+
+export const H1 = styled.h1`
+    font-family: 'Gentium Book Basic', serif;
```
Comment (mikitovicsmark) [inline components/atoms/typography.tsx]: I don't see a lot of duplication here, but in the future it would have some advantages to export some of these stylings in a const, so you would have something like

```
const fontA = `
  font-family: 'a';
  font-weight: 123;
  line-height: 123%;
`

const compA = styled.div`
  ${fontA};
  
  ...
`
```

File: tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    "incremental": true
+  },
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "**/*.mdx",
+  ],
+  "exclude": [
+    // "node_modules"
```
Comment (mikitovicsmark) [inline tsconfig.json]: Is this intentional here ?

File: components/atoms/colors.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+export const Black100 = '#08111D'
```
Comment (mikitovicsmark) [inline components/atoms/colors.ts]: These colors are too ambiguous for me, but i saw that they are named like this in figma too :(

File: .eslintrc.json (hunk 1 above)
Comment (thunderTheGun) [reply .eslintrc.json:8]: Agreed, it is only lazyness from my part

File: tsconfig.json (hunk 5 above)
Comment (thunderTheGun) [reply tsconfig.json]: the whole file is copied from riltech.co tbh

File: components/atoms/colors.ts (hunk 6 above)
Comment (thunderTheGun) [reply components/atoms/colors.ts]: I now tried but couldn't rename them in Figma

Comment (mikitovicsmark) [reply components/atoms/colors.ts]: unfortunate :( if we had something like light/dark/default we could create an object here and add the colors like
blue.light / blue.dark ...

this 100 / 010 is more prone to errors, but nevermind, better keep it consistent

File: tsconfig.json (hunk 5 above)
Comment (Roverr) [reply tsconfig.json]: Should be excluded probably

File: components/atoms/colors.ts (hunk 6 above)
Comment (Roverr) [inline components/atoms/colors.ts]: I think it is questionable if these should qualify as atoms. 

[Take a look at this](https://atomicdesign.bradfrost.com/chapter-2/#atoms)

They describe, how building blocks can look like but they ain't building blocks in themselves. I'd much rather see this file under like `components/styling` or something

File: components/atoms/typography.styles.ts [hunk 7]
```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { Grey010 } from './colors';
+
+export const PageTitle = styled.h1`
```
Comment (Roverr) [inline components/atoms/typography.styles.ts:4]: Now these I'd much rather call building blocks compared to colours

File: pages/index.js [hunk 8]
```diff
@@ -1,6 +1,8 @@
 import Head from 'next/head'
```
Comment (Roverr) [inline pages/index.js]: Should not this file be .tsx ? But at least .jsx, as it contains React syntax

**Review by Roverr — APPROVED** [review]
